src/main/resources/reference.conf

```diff
@@ -25,4 +25,35 @@ spray.can {
     request-timeout = 40s
     connecting-timeout = 40s
   }
-}
\ No newline at end of file
+}
```

Comment:
As per the convo this morning should we move this whole thing to application.conf? I think so - any dissenters?